client/client-multi/client-c/src/kaa/platform-impl/common/kaa_tcp_channel.c

```diff
@@ -1120,39 +1153,44 @@ kaa_error_t kaa_tcp_channel_authorize(kaa_tcp_channel_t *self)
             &sync_buffer,
             &sync_size);
 
```

Comment:
`kaa_platform_protocol_alloc_serialize_client_sync` few lines above returns error code. It should be checked